src/deepsparse/transformers/pipelines/pipeline.py

```diff
@@ -154,6 +153,22 @@ def tokens_to_engine_input(
 
         return [tokens[name] for name in self.onnx_input_names]
 
+    @staticmethod
+    def should_bucket(*args, **kwargs) -> bool:
+        sequence_length = kwargs.get("sequence_length", 128)
+        return isinstance(sequence_length, list)
```

Comment:
what kind of error do we get if we try to pass a list of sequence_lengths to a pipeline that doesn't support bucketing?

Reply:
But added an extra check on BucketingPipeline side if users try to instantiate directly (for safety)